src/main/php/scriptlet/HttpScriptletRequest.class.php

```diff
@@ -281,6 +281,14 @@ public function getSessionId() {
    * @return string
    */
   protected function importParam($value) {
+
+    // Handle file uploads - these are represented as arrays containg elements for each
+    // file being uploaded
```

Comment:
@thekid: your example should work; what would not work is passing in multiple dimensional arrays. That's - in a particular way - what we have with a file upload.